CefSharp/IRequestHandler.cs

```diff
@@ -170,5 +169,14 @@ public interface IRequestHandler
         /// To redirect or retry the resource modify request (url, headers or post body) and return true.
         /// </returns>
         bool OnResourceResponse(IWebBrowser browserControl, IBrowser browser, IFrame frame, IRequest request, IResponse response);
-    }
+
+        /// <summary>
+        /// Invoked when an element in the UI gains focus (or possibly no element gains focus; i.e. an element lost focus).
+        /// </summary>
+        /// <param name="browserControl">The ChromiumWebBrowser control</param>
+        /// <param name="browser">the browser object</param>
+        /// <param name="frame">The frame object</param>
+        /// <param name="node">An object with information about the node that has focus.</param>
+        void OnFocusedNodeChanged (IWebBrowser browserControl, IBrowser browser, IFrame frame, IDomNode node);
+	}
```

Comment:
Formatting is still a little out here. Don't worry about fixing this one as It's unlikely that `OnFocusedNodeChanged` will remain as part of `IRequestHandler`